### DISCUSSION TITLE
issue on using image field with Repeater

### DISCUSSION THREAD

**Discussion by pwlgm1** [laravel/nova-issues#5842] [Ideas & Feature Requests] [answered]
- Laravel Version: v10.19.0
- Nova Version: v4.27.12
- PHP Version: 8.2.9
- Database Driver & Version:
- Operating System and Version: #.#
- Browser type and version: #.#
- Reproduction Repository: https://github.com/###/###

### Description:
When modifying a resource, if you save without adding a file to the image field within the repeater, the existing file will be deleted.

**Comment by crynobone** [comment, marked as answer]
Image extends File field is listed as one of our partially supported fields: https://nova.laravel.com/docs/resources/repeater-fields.html#partially-supported-fields
